### PR TITLE
Semi automate release

### PR DESCRIPTION
**Pull request by yuumasato**
#### Description:

- Scripts to semi-automate release of content.
  - This is heavily inspired by OpenSCAP's release scripts

### CODE REVIEW

File: release_tools/README.adoc [hunk 1]
```diff
@@ -0,0 +1,92 @@
+= How to perform a semi-automatic release =
+
+=== Prerequisities ===
+
+* System with `python3.6` or greater, installed https://pypi.python.org/pypi/PyGithub/1.35[PyGithub] module.
+* A GitHub token - get one on https://github.com/settings/tokens. Choose `repo` scope.
+* Your Jenkins user - your User ID should be the same as your GitHub handle, to be sure, check your profile page in Jenkins.
+* A Jenkins token - gnerate a token on Configure page in Jenkins
```
Comment (mildas) [inline release_tools/README.adoc]: I'd say there's no need for indefinite article `A` in list of items (same for "**A** GitHub token").
+ misspelled `generate` (`gnerate`).

**Comment by redhatrises** [comment]
Can we make this all a python script rather than mixing and matching between bash and python?

**Comment by yuumasato** [comment]
> Can we make this all a python script rather than mixing and matching between bash and python?

Probably, I think the `cmake`, and  `git commands would still require bash snippets. But the python script could run it under the hood.

File: release_tools/release-utils.sh [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+    echo :: Check rhel6-stig-ids.log for rules missing STIG IDs
+    (PYTHONPATH=.. python3 ../build-scripts/profile_tool.py stats -b ../build/ssg-rhel7-xccdf.xml --missing-stig-ids --profile stig) > rhel7-stig-ids.log
+    echo :: Check rhel7-stig-ids.log for rules missing STIG IDs
+
+}
+
+make_dist()
+{
+    if [ -e artifacts/scap-security-guide-$version.tar.bz2 ]; then
+        echo Tarball already exists
```
Comment (jan-cerny) [inline release_tools/release-utils.sh]: If I understand it well this mean there is a leftover file with unknown contents. Do you think it should terminate the script? Or remove the file?

Comment (yuumasato) [reply release_tools/release-utils.sh]: > If I understand it well this mean there is a leftover file with unknown contents.

Not necessarily. `make_dist()` is called by `build.sh`, which can be called multiple times while waiting for builds to finish. This is to check whether the tarball already exists, and avoid repeatedly generating it.

The tarball is removed when `cleanup.sh` is run, whole `artifacts/` directory is removed. See this [line](https://github.com/ComplianceAsCode/content/pull/4666/files#diff-184c7a1349c20ffd1f249962c52075a9R138).

> Do you think it should terminate the script? Or remove the file?

The `make_dist()` command can probably be moved to a different script.
I'll consider this when getting rid of the `.sh` scripts and everything is "pythonized".

**Comment by shawndwells** [comment]
Closing due to approximately 4 months of inactivity. If you'd like to reopen and continue this work, please do!

**Pull request closed without merging**